database/SQL/helpers.js

```diff
@@ -1,3 +1,117 @@
 const db = require('./index.js');
 
-module.exports = {};
\ No newline at end of file
+var getProductList = async (count, page) => {
+	return new Promise (async (resolve, reject) => {
+		var result = [];
+		var start = ((page-1) * count) + 1;
+		for (var i = ((page-1) * count) + 1; i < start + count; i++) {
```

Comment:
The async/await looks professional. I recall trying to use async/await before and I saw that you could use .map, which could replace your for loop here